azurerm/internal/services/databasemigration/tests/resource_arm_database_migration_project_test.go

```diff
@@ -0,0 +1,206 @@
+			},
+			data.ImportStep(),
+		},
+	})
+}
+
+func testCheckAzureRMDatabaseMigrationProjectExists(resourceName string) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		rs, ok := s.RootModule().Resources[resourceName]
+		if !ok {
+			return fmt.Errorf("Database Migration Project not found: %s", resourceName)
+		}
+
+		name := rs.Primary.Attributes["name"]
+		resourceGroup := rs.Primary.Attributes["resource_group_name"]
+		serviceName := rs.Primary.Attributes["service_name"]
+
+		client := acceptance.AzureProvider.Meta().(*clients.Client).DatabaseMigration.ProjectsClient
+		ctx := acceptance.AzureProvider.Meta().(*clients.Client).StopContext
+
+		if resp, err := client.Get(ctx, resourceGroup, serviceName, name); err != nil {
+			if utils.ResponseWasNotFound(resp.Response) {
+				return fmt.Errorf("Bad: Database Migration Project (Project Name %q / Service Name %q / Group Name %q) does not exist", name, serviceName, resourceGroup)
+			}
+			return fmt.Errorf("Bad: Get on ProjectsClient: %+v", err)
+		}
+
+		return nil
+	}
+}
+
+func testCheckAzureRMDatabaseMigrationProjectDestroy(s *terraform.State) error {
+	client := acceptance.AzureProvider.Meta().(*clients.Client).DatabaseMigration.ProjectsClient
+	ctx := acceptance.AzureProvider.Meta().(*clients.Client).StopContext
+
+	for _, rs := range s.RootModule().Resources {
+		if rs.Type != "azurerm_database_migration_project" {
+			continue
+		}
+
+		name := rs.Primary.Attributes["name"]
+		resourceGroup := rs.Primary.Attributes["resource_group_name"]
+		serviceName := rs.Primary.Attributes["service_name"]
+
+		if resp, err := client.Get(ctx, resourceGroup, serviceName, name); err != nil {
+			if !utils.ResponseWasNotFound(resp.Response) {
+				return fmt.Errorf("Bad: Get on ProjectsClient: %+v", err)
+			}
+		}
+
+		return nil
+	}
+
+	return nil
+}
+
+func testAccAzureRMDatabaseMigrationProject_basic(data acceptance.TestData) string {
+	template := testAccAzureRMDatabaseMigrationService_basic(data)
+
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_database_migration_project" "test" {
+	name                = "acctestDbmsProject-%d"
+	service_name        = azurerm_database_migration_service.test.name
+	resource_group_name = azurerm_resource_group.test.name
+	location            = azurerm_resource_group.test.location
+	source_platform     = "SQL"
+	target_platform     = "SQLDB"
+}
+`, template, data.RandomInteger)
+}
+
+func testAccAzureRMDatabaseMigrationProject_complete(data acceptance.TestData) string {
+	template := testAccAzureRMDatabaseMigrationService_basic(data)
+
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_database_migration_project" "test" {
+	name                = "acctestDbmsProject-%d"
+	service_name        = azurerm_database_migration_service.test.name
+	resource_group_name = azurerm_resource_group.test.name
+	location            = azurerm_resource_group.test.location
+	source_platform     = "SQL"
+	target_platform     = "SQLDB"
+    tags = {
+ 		name = "test"
```

Comment:
For tags could we check casing 
```suggestion
 		Name = "Test"
```